phoenix-core/src/main/java/org/apache/phoenix/coprocessor/GroupedAggregateRegionObserver.java

```diff
@@ -402,8 +405,8 @@ private RegionScanner scanUnordered(ObserverContext<RegionCoprocessorEnvironment
             }
 
             Region region = c.getEnvironment().getRegion();
-            region.startRegionOperation();
             try {
+                region.startRegionOperation();
```

Comment:
Doesn't startRegionOperation() take a lock which would prevent a split from occurring and thus a NotServingRegionException from happening? Or is this the case where the split occurred after pre open, but before we start iterating? Let's doc that please.